### ISSUE TITLE
Как решать вопрос согласования ДК в части финансирования?

### ISSUE THREAD

**Issue by Pavel-Bulat** [2035-Platform/live-roadmap-template#23]
В движении НТИ присутствует родовая травма. Авторы стратегии (РГ и Платформа НТИ) не обладают вообще никакими финансовыми ресурсами. При этом ставят цели и задачи, которые на 95% зависят от прямого финансирования и мер финансовой поддержки. Суммы, выделяемые РВК из бюджета Минобрнауки, примерно в 25 раз меньше, чем потребность ДК. Отсюда и следует тенденция не рассматривать конкретные проекты, т.к. денег на системное финансирования, всё-равно, нет.

Тогда единственный выход - это на старте поддерживать какие-то общесистемные мероприятия, типа организации конкурсов, подготовку комплексных программ (КНТП) и т.п., а на финише подключать ВЭБ и инвестировать на стадии, когда НИОРКР завершены уже. 

В промежутке компании и РГ должны идти в ФОИВы, но их там должны ждать, они должны вписываться в систему финансирования. Следовательно, по идее, в части финансирования ДК должны быть согласованы. Но это невозможно. 

Например, вы хотим согласовать финсирование в Минобре в форме КНТП. Если мы идем туда на общих основаниях, то НТИ не нужно вообще. Если мы идем туда по программе НТИ, то нужна согласованная ДК с тематикой исследования. Получается замкнутый круг. 

Аналогичная ситуация с Минпромторгом. Они могут финансировать НИР силами консорциума, но все деньги заранее расписываются по внутренним программам. Опять же, если мы согласовываем на уровне профильного ФОИВа тематику НИР и ОКР, например, как в декабрьском 2019 г Постановлении Правительства по консорциумам в области бортового оборудования, то зачем нам вообще НТИ? 

С другой стороны, в целевых показателях должна быть жесткая привязка к уровню бюджетного финансирования, иначе получатся, что Аэронет профинансировали на 11%, а требуют 100% достижения плановых показатей.

**Comment by ilia-metreveli** [comment]
Согласен. Но это не вопрос структуры ДК, наверное. Но вопрос исключительно важный. 
Возможно, увязать НТИ с Нац.проектами было бы выходом, на уровне конкретных предложений я затрудняюсь что-то сказать.

**Comment by kichinsky** [comment]
Имеет ли смысл такую "связку" заводить в саму ДК в части анализа, например? В текущем формате по 317ПП есть раздел про 
> Сведения о документах стратегического планирования, относящихся к категории разрабатываемых на федеральном уровне, по отраслевому и территориальному принципу, а также в рамках прогнозирования, положения которых учтены при разработке плана мероприятий ("дорожной карты").

**Comment by Pavel-Bulat** [comment]
Я думаю, что да
 
 
Павел Викторович Булат

**Comment by ilia-metreveli** [comment]
> Я думаю, что да     Павел Викторович Булат    
> Четверг, 11 июня 2020, 0:43 +03:00 от Konstantin Kichinsky ***@***.***>:     Имеет ли смысл такую "связку" заводить в саму ДК в части анализа, например? В текущем формате по 317ПП есть раздел про >Сведения о документах стратегического планирования, относящихся к категории разрабатываемых на федеральном уровне, по отраслевому и территориальному принципу, а также в рамках прогнозирования, положения которых учтены при разработке плана мероприятий ("дорожной карты"). — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub , or unsubscribe .

Возможно, но процесс должен быть синхронный. Если мы в НТИ учитываем НП, а они - нет, тогда нет смысла. Были гос.задания по темам НТИ. Мне кажется, был хороший пример.

**Comment by Pavel-Bulat** [comment]
Это был единственный удачный пример финансирования НТИ.
